packages/loaders/src/Progress.spec.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import Progress from './Progress';
+const { zdColorBlue600 } = require('@zendeskgarden/css-variables');
+
+describe('Progress', () => {
+  describe('without a value', () => {
+    it('renders a progress bar with 0% progress', () => {
+      const { getByRole } = render(<Progress />);
+
+      expect(getByRole('progressbar')).toHaveAttribute('aria-valuenow', '0');
+    });
+  });
+
+  it('renders a progress bar with correct aria attributes', () => {
+    const { getByRole } = render(<Progress value={40} />);
+
+    const progress = getByRole('progressbar');
+
+    expect(progress).toHaveAttribute('role', 'progressbar');
```

Comment:
If you would like, you can remove this assertion because the one on lines 25 will implicitly verify that a role of `progressbar` exists. Not a big deal or a blocker though.